### PR TITLE
Made a button that brings up the open file dialog

### PR DESCRIPTION
**Pull request by Salamander388**

### CODE REVIEW

File: ValheimServer/ValheimServer/MainWindow.xaml.cs [hunk 1]
```diff
@@ -21,11 +22,18 @@ namespace ValheimServer
...
     public partial class MainWindow : Window
     {
+
         public MainWindow()
         {
             InitializeComponent();
         }
 
-     
+        //I dont know where I am supposed to stick anything so im putting this here
```
Comment (Sacharr) [inline ValheimServer/ValheimServer/MainWindow.xaml.cs:31]: Move this code inside the body of your ofdButton_Click method.

```
private void ofdButton_Click(object sender, RoutedEventArgs e)
        {
           OpenFileDialog ofd = new OpenFileDialog();

           ofd.ShowDialog();
        }
```

File: ValheimServer/ValheimServer/MainWindow.xaml.cs [hunk 2]
```diff
@@ -21,11 +22,18 @@ namespace ValheimServer
...
+        OpenFileDialog ofd = new OpenFileDialog();
+
+        private void ofdButton_Click(object sender, RoutedEventArgs e)
+        {
+            ofd.ShowDialog();
```
Comment (Sacharr) [inline ValheimServer/ValheimServer/MainWindow.xaml.cs:36]: Do not abbreviate variable names either.

File: ValheimServer/ValheimServer/MainWindow.xaml [hunk 3]
```diff
@@ -11,7 +11,17 @@
...
     <Grid>
-        <Button x:Name="SteamcmdButton" Content="{Binding SteamButtonText, Mode=OneWay}" HorizontalAlignment="Left" Margin="67,78,0,0" VerticalAlignment="Top" Height="34" Width="116" Command="{Binding SteamButtonClickCommand}"/>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition/>
+        </Grid.ColumnDefinitions>
+        <Button x:Name="SteamcmdButton" Content="{Binding SteamButtonText, Mode=OneWay}" 
+                HorizontalAlignment="Left" Margin="67,78,0,0" VerticalAlignment="Top" Height="34" Width="116" 
+                Command="{Binding SteamButtonClickCommand}"/>
+
+        <Button x:Name="ofdButton" Content="OpenFileDialog" HorizontalAlignment="Right" Margin="0,77,48,0"
```
Comment (Sacharr) [inline ValheimServer/ValheimServer/MainWindow.xaml:23]: Change the abbreviated name.

**Review by Sacharr — CHANGES_REQUESTED** [review]

**Comment by Salamander388** [comment]
@Sacharr I made the requested changes.

**Review by Sacharr — APPROVED** [review]

**Pull request merged**